app/main.py

```diff
@@ -1,6 +1,27 @@
+import requests
+from decouple import config
+
+
+API_KEY = config("API_KEY")
+CITY_NAME = "Paris"
+URL = (f"https://api.weatherapi.com/v1/current.json"
+       f"?key={API_KEY}&q={CITY_NAME}&aqi=no")
```

Comment:
it is bad practice to use f-string for url params, use `params` param in requests.get method instead